config/common/errors.go

```diff
@@ -49,8 +49,9 @@ var (
 	ErrTooManySystemdSources = errors.New("only one of the following can be set: contents, contents_local")
 
 	// mount units
-	ErrMountUnitNoPath   = errors.New("path is required if with_mount_unit is true and format is not swap")
-	ErrMountUnitNoFormat = errors.New("format is required if with_mount_unit is true")
+	ErrMountUnitNoPath     = errors.New("path is required if with_mount_unit is true and format is not swap")
+	ErrMountUnitNoFormat   = errors.New("format is required if with_mount_unit is true")
+	ErrMountPointForbidden = errors.New("forbidden mount point, Use a mount point that starts with /etc or /var")
```

Comment:
This block is currently for mount unit generation (`with_mount_unit: true`), and none of the other blocks seem to fit.  Let's either start a new block or update the header comment for this one.

Maybe we could tighten up the error text e.g. like this?
```suggestion
	ErrMountPointForbidden = errors.New("mounts must be under /etc or /var")
```